show/bgp_frr_v4.py

```diff
@@ -24,43 +24,93 @@ def bgp():
 @bgp.command()
 @multi_asic_util.multi_asic_click_options
 def summary(namespace, display):
-    bgp_summary = bgp_util.get_bgp_summary_from_all_bgp_instances(constants.IPV4, namespace,display)
+    bgp_summary = bgp_util.get_bgp_summary_from_all_bgp_instances(
+        constants.IPV4, namespace, display)
     bgp_util.display_bgp_summary(bgp_summary=bgp_summary, af=constants.IPV4)
-    
+
 
 # 'neighbors' subcommand ("show ip bgp neighbors")
 @bgp.command()
 @click.argument('ipaddress', required=False)
-@click.argument('info_type', type=click.Choice(['routes', 'advertised-routes', 'received-routes']), required=False)
-def neighbors(ipaddress, info_type):
+@click.argument('info_type',
+                type=click.Choice(
+                    ['routes', 'advertised-routes', 'received-routes']),
+                required=False)
+@click.option('--namespace',
+                '-n',
+                'namespace',
+                default=None,
+                type=str,
+                show_default=True,
+                help='Namespace name or all',
+                callback=multi_asic_util.multi_asic_namespace_validation_callback)
```

Comment:
why not add '-d' option to display only the external BGP sessions. Would be good to follow the CLI norm we've been following to display the frontend results by default.

Reply:
@smaheshm, as discussed offline adding `-d` option to this command adds lot of complexity and/or slowness to the cli command.  The command will display all the neighbor information, the `ipaddress` option can be used to display information for specific neighbor.